RetroFE/Source/Execute/Launcher.cpp

```diff
@@ -73,7 +73,7 @@ bool Launcher::run(std::string collection, Item *collectionItem, Page *currentPa
     }
 
     // Convert launcherName to lowercase for consistency
-    //launcherName = Utils::toLower(launcherName);
+   //launcherName = Utils::toLower(launcherName);
```

Comment:
I didn't look closely but does this cause mismatch now with above removal of lower casing?